src/main/resources/db_scripts/postgres-rutherford-create-script.sql

```diff
@@ -109,6 +109,20 @@ ALTER TABLE public.event_bookings_id_seq OWNER TO rutherford;
 ALTER SEQUENCE public.event_bookings_id_seq OWNED BY public.event_bookings.id;
 
 
+--
+-- Name: external_accounts; Type: TABLE; Schema: public; Owner: rutherford
+--
+
+CREATE TABLE public.external_accounts (
+    user_id integer NOT NULL,
+    provider_name text NOT NULL,
+    provider_user_identifier text,
+    provider_last_updated timestamp without time zone
+);
```

Comment:
This looks very similar to the thirdparty authenticators table. I wonder if in the future they could be combined or if we think that is a bad idea. Party of my likes to keep authentication stuff nice and separate but it does seem to be fulfilling the same purpose - i.e. linking an isaac user to another identity elsewhere

Reply:
I thought that too, but the code for SSO providers assumes all rows in this table correspond to valid `AuthenticationProvider` enum values, and trying to rewrite all that code seemed much too much effort.